.github/workflows/ci.yml

```diff
@@ -83,6 +100,23 @@ jobs:
               --expected-major-compiler-version=17
               --lto=full
 
+          - name: centos7-aarch64-clang17
```

Comment:
The centos7 entries should be removed. It is EOL and we no longer support it on any release. The amzn2 OS should be used as the lowest version glibc to be compatible with.